{
    "Config_Baseline": {
        "ERROR_MAIN_LOOP": "sklearn needs to be installed in order to use this module",
        "traceback": "Traceback (most recent call last):\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_accuracy.py\", line 308, in <module>\n    results_for_this_config = run_all_tests_for_config(config_params, verbose=verbose_output)\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_accuracy.py\", line 233, in run_all_tests_for_config\n    results_xgb = run_xgboost_peer_test(\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_harness.py\", line 460, in run_xgboost_peer_test\n    model = xgb.XGBRegressor(**xgboost_params)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/core.py\", line 726, in inner_f\n    return func(**kwargs)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/sklearn.py\", line 1794, in __init__\n    super().__init__(objective=objective, **kwargs)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/sklearn.py\", line 744, in __init__\n    raise ImportError(\nImportError: sklearn needs to be installed in order to use this module\n"
    },
    "Config_Strict_Alpha": {
        "ERROR_MAIN_LOOP": "sklearn needs to be installed in order to use this module",
        "traceback": "Traceback (most recent call last):\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_accuracy.py\", line 308, in <module>\n    results_for_this_config = run_all_tests_for_config(config_params, verbose=verbose_output)\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_accuracy.py\", line 233, in run_all_tests_for_config\n    results_xgb = run_xgboost_peer_test(\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_harness.py\", line 460, in run_xgboost_peer_test\n    model = xgb.XGBRegressor(**xgboost_params)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/core.py\", line 726, in inner_f\n    return func(**kwargs)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/sklearn.py\", line 1794, in __init__\n    super().__init__(objective=objective, **kwargs)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/sklearn.py\", line 744, in __init__\n    raise ImportError(\nImportError: sklearn needs to be installed in order to use this module\n"
    },
    "Config_Loose_Alpha": {
        "ERROR_MAIN_LOOP": "sklearn needs to be installed in order to use this module",
        "traceback": "Traceback (most recent call last):\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_accuracy.py\", line 308, in <module>\n    results_for_this_config = run_all_tests_for_config(config_params, verbose=verbose_output)\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_accuracy.py\", line 233, in run_all_tests_for_config\n    results_xgb = run_xgboost_peer_test(\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_harness.py\", line 460, in run_xgboost_peer_test\n    model = xgb.XGBRegressor(**xgboost_params)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/core.py\", line 726, in inner_f\n    return func(**kwargs)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/sklearn.py\", line 1794, in __init__\n    super().__init__(objective=objective, **kwargs)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/sklearn.py\", line 744, in __init__\n    raise ImportError(\nImportError: sklearn needs to be installed in order to use this module\n"
    },
    "Config_Shallow_Tree": {
        "ERROR_MAIN_LOOP": "sklearn needs to be installed in order to use this module",
        "traceback": "Traceback (most recent call last):\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_accuracy.py\", line 308, in <module>\n    results_for_this_config = run_all_tests_for_config(config_params, verbose=verbose_output)\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_accuracy.py\", line 233, in run_all_tests_for_config\n    results_xgb = run_xgboost_peer_test(\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_harness.py\", line 460, in run_xgboost_peer_test\n    model = xgb.XGBRegressor(**xgboost_params)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/core.py\", line 726, in inner_f\n    return func(**kwargs)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/sklearn.py\", line 1794, in __init__\n    super().__init__(objective=objective, **kwargs)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/sklearn.py\", line 744, in __init__\n    raise ImportError(\nImportError: sklearn needs to be installed in order to use this module\n"
    },
    "Config_High_Min_Samples": {
        "ERROR_MAIN_LOOP": "sklearn needs to be installed in order to use this module",
        "traceback": "Traceback (most recent call last):\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_accuracy.py\", line 308, in <module>\n    results_for_this_config = run_all_tests_for_config(config_params, verbose=verbose_output)\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_accuracy.py\", line 233, in run_all_tests_for_config\n    results_xgb = run_xgboost_peer_test(\n  File \"/Users/kevinkoning/Library/CloudStorage/GoogleDrive-kevin.koning@gmail.com/My Drive/Code/BinomialTree/tests/test_harness.py\", line 460, in run_xgboost_peer_test\n    model = xgb.XGBRegressor(**xgboost_params)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/core.py\", line 726, in inner_f\n    return func(**kwargs)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/sklearn.py\", line 1794, in __init__\n    super().__init__(objective=objective, **kwargs)\n  File \"/Users/kevinkoning/Library/Python/3.9/lib/python/site-packages/xgboost/sklearn.py\", line 744, in __init__\n    raise ImportError(\nImportError: sklearn needs to be installed in order to use this module\n"
    }
}